packages/webamp-modern-2/src/skin/parse.ts

```diff
@@ -248,9 +319,10 @@ export default class SkinParser {
       "Unexpected children in <script> XML node."
     );
 
-    const { file, id } = node.attributes;
+    let { file, param } = node.attributes;
     assert(file != null, "Script element missing `file` attribute");
     // assert(id != null, "Script element missing `id` attribute");
+    file = file.replace("../Winamp Modern/", "");
```

Comment:
this may have been from trying to get my fork of winamp modern is referencing the winamp modern folder to explicitly look in there (as it makes sharing files with the included holiday edition easier), like here
`<bitmap id="player.main.left" file="../Winamp Modern/window/main.png" x="0" y="0" h="126" w="180" gammagroup="Backgrounds"/>`

Reply:
oh, it was my quick and dirty patch to work with @0x5066 (one of his modern winamp skin).
I think it is good to reject it now? and **reallow it later** when we could load skin from a folder.